demo/directives.html

```diff
@@ -48,24 +48,24 @@ <h4 class="lui underline">Supported arguments:</h4>
 		</div>
 		<p>
 			<ul class="lui unstyled">
-				<li><code>ng-model</code>: the variable to bind to</li>
-				<li><code>ng-change</code>: events to launch when the value is updated</li>
-				<li><code>ng-disabled</code>: </li>
-				<li><code>unit</code>: 'hour' or 'minute', tells the directive that typing 10 in the input means 10 minutes or 10 hours. The default is 'minute'</li>
-				<li><code>step</code>: when scrolling or pressing the up/down key, how many minutes you want to be added. The default is 5</li>
-				<li><code>placeholder</code>: placeholder to display when the input is empty</li>
-				<li><code>mode</code>: to tell if you want the directive to give you a timespan or a <a href="http://momentjs.com/docs/#/durations/" class="lui fancy link">moment.duration</a>. Supported values: <code>timespan</code> (default) and <code>moment.duration</code></li>
+				<li><code>ng-model</code> : the variable to bind to</li>
```

Comment:
in english there are no spaces between a word and a colon

_in french_

> Lorem : ipsum dolor sit amet

_in english_

> Lorem: ipsum dolor sit amet